packages/ubuntu_desktop_installer/lib/pages/timezone/timezone_page.dart

```diff
@@ -12,38 +12,26 @@ import 'package:yaru_widgets/yaru_widgets.dart';
 import 'timezone_model.dart';
 
 /// https://github.com/canonical/ubuntu-desktop-installer/issues/38
-class TimezonePage extends StatefulWidget {
-  /// Use [TimezonePage.create] instead.
-  @visibleForTesting
+class TimezonePage extends ConsumerStatefulWidget {
   const TimezonePage({super.key});
 
-  /// Creates a [TimezonePage] with [TimezoneModel].
-  static Widget create(BuildContext context) {
-    return MultiProvider(
-      providers: [
-        ChangeNotifierProvider(
-          create: (_) => TimezoneModel(getService<SubiquityClient>()),
-        ),
-        ChangeNotifierProvider(
-          create: (_) => TimezoneController(service: getService<GeoService>()),
-        ),
-      ],
-      child: const TimezonePage(),
-    );
-  }
+  static final timezoneModelProvider = ChangeNotifierProvider(
```

Comment:
`TimezonePage.timezoneModelProvider` gets quite long and seems repetitive, is the "timezone" prefix intentional or did it slip in by accident? :)

Reply:
That wasn't intentional - I renamed it :)